docs/getting_started/installation.rst

```diff
@@ -44,7 +33,7 @@ If you don't have Ray_ or Dask_ installed, you will need to install Modin with o
   pip install modin[all] # Install all of the above
 
 Modin will automatically detect which engine you have installed and use that for
-scheduling computation!
+scheduling computation! See below for OmniSci engine installation
```

Comment:
```suggestion
scheduling computation! See below for OmniSci engine installation.
```
Nit: Add period. 
It might be useful to also specify `See the below "Installing specific dependency sets" section for OmniSci engine installation.`